src/test/regress/multi_schedule

```diff
@@ -29,7 +29,7 @@ test: multi_stage_data
 # ----------
 # Miscellaneous tests to check our query planning behavior
 # ----------
-test: multi_basic_queries multi_complex_expressions multi_verify_no_subquery
+test: multi_basic_queries multi_verify_no_subquery multi_complex_expressions
```

Comment:
Oh you are correct, after removing the multi_complex_expressions from the parallel group and re-adding it, I forgot to consider their previous order.